algobot/interface/configuration.py

```diff
@@ -282,26 +282,26 @@ def create_take_profit_inputs(self, tab: QTabWidget, innerLayout: QLayout, isOpt
         if isOptimizer:
             self.takeProfitDict['optimizerTypes'] = []
             innerLayout.addRow(QLabel("Take Profit Types"))
-            for takeProfitType in self.takeProfitTypes:
-                checkbox = QCheckBox(f'Enable {takeProfitType} take profit?')
+            for takeOrderType in self.takeOrderTypes:
+                checkbox = QCheckBox(f'Enable {takeOrderType} take profit?')
```

Comment:
notice that this is takeProfit

Reply:
It wasn't meant to refactor so much, whoops :D